gas.json is not included in pypi package

I got this error:
```
raiden/network/proxies/token_network.py:170: in __init__
    self.gas_measurements = gas_measurements(self.contract_manager.contracts_version)
../../.pyenv/versions/3.7.0/envs/raiden/lib/python3.7/site-packages/raiden_contracts/contract_manager.py:151: in gas_measurements
    with json_path.open() as gas_file:
../../.pyenv/versions/3.7.0/lib/python3.7/pathlib.py:1165: in open
    opener=self._opener)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   return self._accessor.open(self, flags, mode)
E   FileNotFoundError: [Errno 2] No such file or directory: '/home/rakan/.pyenv/versions/3.7.0/envs/raiden/lib/python3.7/site-packages/raiden_contracts/data_0.14.0/gas.json'
```

When i investigated, i found that `MANIFEST.in` does not include gas.json to be included in the pypi package, therefore, i am seeing this failure.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -11,6 +11,7 @@ include raiden_contracts/data*/deployment_services_kovan.json
 include raiden_contracts/data*/deployment_services_goerli.json
 include raiden_contracts/data*/deployment_services_rinkeby.json
 include raiden_contracts/data*/deployment_services_ropsten.json
+include raiden_contracts/data*/gas.json
 recursive-include raiden_contracts/data*/source *
 prune raiden_contracts/contracts
 prune raiden_contracts/contracts_without_limits

